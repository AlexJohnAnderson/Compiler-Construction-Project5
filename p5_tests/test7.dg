void func(int a){
	int b;
	b = a * 2;
	return b;
}

int main(){
	bool param;
	param = true;
	func(param);
}

	