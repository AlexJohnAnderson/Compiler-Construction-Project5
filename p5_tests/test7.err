FATAL [4,9]-[4,10]: Return with a value in void function
FATAL [10,2]-[10,6]: Type of actual does not match type of formal
Type Analysis Failed
