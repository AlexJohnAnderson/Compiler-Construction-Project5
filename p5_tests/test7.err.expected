FATAL [4,9]-[4,10]: Return with a value in void function
FATAL [10,7]-[10,12]: Type of actual does not match type of formal
Type Analysis Failed
